src/js/password-strength.js

```diff
@@ -0,0 +1,30 @@
+import { Component } from 'react';

```

Comment:
src/js/password-strength.js is the only file that is missing -view` suffix with the name. fix it